test/onnx/test_pytorch_onnx_caffe2.py

```diff
@@ -2438,5 +2438,89 @@ def setup_rnn_tests():
                                            embed_params=True, opset_version=10))
 
 
+class TestQuantizedOps(unittest.TestCase):
```

Comment:
Actually, why not create a test_pytorch_onnx_quantization_caffe2.py? I didn't see you reuse anything in this file.

Reply:
Done